README.md

```diff
@@ -450,15 +450,16 @@ We do maintain a [roadmap regarding next releases of this module](ROADMAP.md).
 
 ### Operating System support matrix
 
-| OS          | release | Puppet 3.8.7    | Puppet 4.X (PC1) |
-|-------------|---------|---------------|------------------|
-| CentOS/RHEL | 5       | Not supported | Not supported    |
-| CentOS/RHEL | 6       | Not supported | Not supported    |
-| CentOS/RHEL | 7       | **Supported** | **Supported**    |
-| Debian      | 8       | Not supported | **Supported[1]** |
-| Ubuntu      | 12.04   | Not supported | Not supported    |
-| Ubuntu      | 14.04   | **Supported** | **Supported**    |
-| Ubuntu      | 16.04   | Not supported | **Supported**    |
+| OS          | release | Puppet 3.8.7  | Puppet 4.X (PC1) | Puppet 5.X       |
```

Comment:
In metadata we list 4.10 as the minimum puppet version.

Reply:
PC1 being puppet 4.10.x, do you prefer "Puppet 4 (PC1)" ?

Reply:
I think that'd be better